bedrock/settings/base.py

```diff
@@ -1256,7 +1256,8 @@ def before_send(event, hint):
     "logs.convertexperiments.com",
     "1003350.metrics.convertexperiments.com",
     "1003343.metrics.convertexperiments.com",
-    "sentry.prod.mozaws.net",
+    "sentry.prod.mozaws.net",  # DEPRECATED. TODO: remove this once all sites are talking to sentry.io instead
+    "*.sentry.io",
```

Comment:
@alexgibson @pmac What do you thikn about using a wildcard subdomain here? 

Thinking about it, we could def pull it from config instead (eg split it off the value of `SENTRY_DSN`), but wondering if you think it's worth it

Reply:
Parallel conversation going on in Slack. Gonna make it explicit, yeah 👍